lib/ColumnManager/ColumnManager.js

```diff
@@ -75,19 +21,18 @@ const ColumnManager = ({ id, columnMapping, children, excludeKeys }) => {
    * an easy way for developers to render a column menu section inside e.g. a dropdown
    */
   const renderColumnsMenu = useMemo(() => (
-    <MenuSection
-      data-test-column-manager-menu
-      label={intl.formatMessage({ id: 'stripes-smart-components.columnManager.showColumns' })}
-      id={`${prefixId}-columns-menu-section`}
-    >
-      {renderCheckboxes}
-    </MenuSection>
-  ), [renderCheckboxes, intl, prefixId]);
+    <ColumnManagerMenu
+      prefix={prefixId}
+      columnMapping={columnMapping}
+      visibleColumns={visibleColumns}
+      excludeColumns={nonToggleableColumns}
+      toggleColumn={toggleColumn}
+    />
+  ), [prefixId, columnMapping, visibleColumns, nonToggleableColumns, toggleColumn]);
 
   const renderProps = {
-    visibleColumns: orderedVisibleColumns,
+    visibleColumns,
     renderColumnsMenu,
-    renderCheckboxes,
```

Comment:
as I see nobody uses this, and it looks like breaking change, should we update major version now?

Reply:
@zburke please advice

Reply:
I'm inclined not to issue a breaking change for this. 

If you think it is worth removing because it isn't being used, let's  label it deprecated, issue a console warning, and remove it in the next release cycle that contains other breaking changes. A major release is disruptive. I don't think it's worth the churn just to formally remove something that, as you said, perhaps nobody is using anyway. 

How does that sound to you?

Reply:
sounds good to me, I'll try to reuse existing code to not introduce smth else for this prop